modules/ingester/ingester.go

```diff
@@ -239,15 +239,18 @@ func (i *Ingester) PushBytesV2(ctx context.Context, req *tempopb.PushBytesReques
 
 	instance, err := i.getOrCreateInstance(instanceID)
 	if err != nil {
+		level.Warn(log.Logger).Log(err.Error())
 		return nil, err
 	}
 
-	err = instance.PushBytesRequest(ctx, req)
-	if err != nil {
-		return nil, err
+	response, _ := instance.PushBytesRequest(ctx, req)
+	// distributor will determine if an error should be returned to the client
```

Comment:
i think we should continue returning error. any unexpected error in the path should just make the whole thing 500. previously we treated drops due to limits as an error, but this PR changes that.

- everything works perfectly: 200/OK
- nothing fails, but we drop spans: 200/OK and tick up our metrics
- unexpected error: 500/INTERNAL